common/utils/include/claragenomics/utils/device_buffer.cuh

```diff
@@ -68,6 +68,11 @@ public:
         cudaFree(data_);
     }
 
+    void initialize(cudaStream_t stream)
```

Comment:
Passing a stream should be enough to indicate that this is an async operation, but it feels like `initialize_async` would be it even clearer.

Reply:
yes, makes sense. I changed it from `initialize(stream)` to `initialize_async(stream)`. Thanks!